support-frontend/assets/pages/[countryGroupId]/components/headerCards.tsx

```diff
@@ -0,0 +1,124 @@
+import { css } from '@emotion/react';
+import {
+	from,
+	headlineBold24,
+	palette,
+	space,
+	textSans15,
+	textSans17,
+} from '@guardian/source/foundations';
+import { simpleFormatAmount } from 'helpers/forms/checkouts';
+import { currencies } from 'helpers/internationalisation/currency';
+import {
+	productCatalog,
+	productCatalogGuardianLight,
+} from 'helpers/productCatalog';
+import type { GeoId } from 'pages/geoIdConfig';
+import { getGeoIdConfig } from 'pages/geoIdConfig';
+import { ComponentContainer } from './componentContainer';
+import { GuardianLightCards } from './guardianLightCards';
+
+const containerCards = css`
+	background-color: ${palette.brand[400]};
+	border-bottom: 1px solid ${palette.brand[600]};
+`;
+const containerSignIn = css`
+	background-color: ${palette.brand[400]};
+	${from.tablet} {
+		background-color: ${palette.neutral[97]};
+	}
+`;
+const heading = css`
+	color: ${palette.neutral[100]};
+	text-align: left;
+	${headlineBold24}
+	margin-bottom: ${space[6]}px;
+	${from.tablet} {
+		text-align: center;
+	}
+	${from.desktop} {
+		font-size: 2.625rem;
+		margin-bottom: ${space[10]}px;
+	}
+`;
+const signIn = css`
+	color: ${palette.neutral[100]};
+	text-align: left;
+	${textSans15}
+	padding: 0px 0px ${space[1]}px;
+	> a {
+		color: ${palette.neutral[100]};
+		font-weight: bold;
+	}
+	${from.tablet} {
+		color: ${palette.neutral[7]};
+		padding: ${space[3]}px 0px ${space[4]}px;
+		text-align: center;
+		> a {
+			color: ${palette.brand[500]};
+		}
+	}
+	${from.desktop} {
+		${textSans17}
+	}
+`;
+
+const SignInUrl = 'https://manage.theguardian.com/signin';
```

Comment:
Should this vary based on the stage? (I.e. `manage.code.dev-theguardian.com/signin` if we're in code?)

Reply:
I think this (hard-coded) pattern has been used throughout the code-base. Can see would be useful for Code, so have changed for Code but routed Prod/Dev to Prod link  (didnt think Dev would typically have manage running on dev). Hope thats ok.